test/AsDispatcher/FreeReactAsDispatcher.ts

```diff
@@ -0,0 +1,91 @@
+import {expect} from "chai";
+import {ethers} from "hardhat";
+import {deploySystem, TEST_SALE_CREATOR_BP} from "../Scripts/setup";
+import {BigNumber} from "ethers";
+import {
+  deriveLikeTokenIndex,
+  deriveTransformId,
+} from "../Scripts/derivedParams";
+import {ZERO_ADDRESS} from "../Scripts/constants";
+
+describe("ReactionVault FreeReactAsDispatcher", function () {
+  it("Should issue a like token", async function () {
```

Comment:
Might want to do a negative test and "like" before adding the dispatcher.

Reply:
Done!